axelor-account/src/main/java/com/axelor/apps/account/service/analytic/AnalyticLineService.java

```diff
@@ -39,11 +40,16 @@ public interface AnalyticLineService {
   List<Long> getAxisDomains(AnalyticLine line, Company company, int position)
       throws AxelorException;
 
+  List<Long> getAnalyticAccountIdList(Company company, int position) throws AxelorException;
+
   boolean isAxisRequired(AnalyticLine line, Company company, int position) throws AxelorException;
 
   AnalyticLine checkAnalyticLineForAxis(AnalyticLine line);
 
   AnalyticLine printAnalyticAccount(AnalyticLine line, Company company) throws AxelorException;
 
   List<Long> getAnalyticAccountsByAxis(AnalyticLine line, AnalyticAxis analyticAxis);
+
+  Map<String, Map<String, Object>> getAnalyticAxisAttrsMap(
+      Company company, int startAxisPosition, int endAxisPosition) throws AxelorException;
```

Comment:
Same as before, but maybe in a `AnalyticAttrsService` or something like that
Also, its content is similar to what is done on moves for example. Can you check if it can be factorized ?